cmd/registry/cmd/upload-bulk-discovery.go

```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/apigee/registry/rpc"
 	rpcpb "github.com/apigee/registry/rpc"
 	discovery "github.com/googleapis/gnostic/discovery"
+	"github.com/nsf/jsondiff"
```

Comment:
Would this also show up in go.mod? Let's include at least that new line and discuss any other differences that seem worth including

Reply:
Fixed. I just ran `make lite` and let it decide which changes to include.

Reply:
Thanks, I see we picked up a few extra things - we can tighten up expectations about that going forward.